content/en/account_management/teams.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Teams
+kind: documentation
+---
+
+## Overview
+Datadog Teams allow groups of users to organize their team assets within Datadog and automatically filter their Datadog-wide experience to prioritize these assets.
+
+Use Teams to link resources such as dashboards, services, monitors, and incidents to a group of users. You can also add team-specific links to Slack channels, Jira boards, and Github repositories.
```

Comment:
```suggestion
Use Teams to link resources such as dashboards, services, monitors, and incidents to a group of users. You can also add team-specific links to Slack channels, Jira boards, and GitHub repositories.
```

it sounds like these are just links that could be added anywhere — should we end this be "Slack channels, Jira boards, GitHub repositories, etc."?

Reply:
yes, you're right, it does need the "etc."